pt-BR/mail/chrome/messenger/preferences/display.dtd

```diff
@@ -29,64 +29,64 @@
 <!ENTITY removeTagButton.accesskey "x">
 
 <!-- Fonts -->
-<!ENTITY fontsAndColors.label    "Fonts">
-<!ENTITY defaultFont.label       "Default font:">
+<!ENTITY fontsAndColors.label    "Fontes">
+<!ENTITY defaultFont.label       "Fonte padrão:">
 <!ENTITY defaultFont.accesskey   "D">
-<!ENTITY defaultSize.label       "Size:">
+<!ENTITY defaultSize.label       "Tamanho:">
 <!ENTITY defaultSize.accesskey   "S">
 <!ENTITY fontOptions.accesskey   "A">
 <!ENTITY fontOptions.label       "Avançado…">
 
-<!ENTITY pb.messagelist.colortagrow.label "Apply the topic color to the entire row in the message list pane">
+<!ENTITY pb.messagelist.colortagrow.label "Aplique a cor na linha inteira no painel de lista de mensagens">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pb.messagelist.colortagrow.label "Aplicar a mesma cor do tópico, a todas as mensagens presentes na mesma linha do painel de mensagens">